contracts/PetPark.sol

```diff
@@ -1,7 +1,92 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
-
+import "hardhat/console.sol";
 
 contract PetPark {
+    address public owner;
+
+    mapping(string => uint) public AnimalType;
```

Comment:
You can use an enum for this, since mapping here is very storage intensive

Reply:
Changes _AnimalType_ and _Gender_ mappings to _animalTypes_ and _genders_ enums.